Scripts/run-ios-tests.sh

```diff
@@ -0,0 +1,6 @@
+set -o pipefail
+
+xcodebuild clean test \
+-scheme RestKit \
+-destination "platform=iOS Simulator,name=iPhone X,OS=12.1" | xcpretty || RC=${RC:~$?}\
```

Comment:
Minor point: I don't think you need `|| RC=${RC:~$?}`. That is used in the Swift SDK to accumulate the return code across multiple calls to xcodebuild, but here we only build once, so it isn't needed.

Reply:
Ok I removed that block. I was wondering what that did :)